<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ekranLogowania</class>
 <widget class="QDialog" name="ekranLogowania">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super TIN-TIN</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Login:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="textLogin"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Hasło:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="textPass"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushZaloguj">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zaloguj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushZakoncz">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushRejestruj">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rejestracja</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
